tests/foreman/cli/test_http_proxy.py

```diff
@@ -163,29 +185,6 @@ def test_positive_set_content_default_http_proxy(block_fake_repo_access, target_
     assert rpm_repo.read().content_counts['rpm'] >= 1
 
 
-@pytest.mark.stubbed
-@pytest.mark.tier3
-def test_positive_environment_variable_unset_set():
-    """Verify that satellite installer unsets and then sets back the environment variables
-
-    :id: 596d753b-660b-49cb-b663-ff3cec439564
-
-    :BZ: 1886040
-
-    :customerscenario: true
-
```

Comment:
I remember this test, its related to old feature added in installer, which identifies and fails, if proxy is set on system, with this mentioned environment variable. So, I understand that this test is valid, but can I ask why are you considering removing it instead of automating it?

Reply:
I considered removing because the BZ says
```
3. Why does the customer need this? (List the business requirements here)

Since satellite maintain takes care of end to end upgrade, it should also have an additional check to unset an environment variable along with alerting the user.
The addition of this check will ensure that we can avoid the installer execution failure and fixing a potential issue at the pre-upgrade stage.
```
This coverage already exists [here](https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/maintain/test_health.py#L542), so I considered it redundant.

Reply:
I don't see it as redundant as the test you linked is being verified by foreman-maintain check which verifies it it exists in the environment, and here we use the installer to test, where behaviour here is different, installers unsets the variable while running and then sets it back when installer run completes

Reply:
Well, my assumptions were following:
1. The foreman-maintain health check (covered by the test linked above) checks if HTTP_PROXY env var is set and fails if it is.
2. Satellite upgrade leverages the foremain-maintain health check as pre-upgrade check.
3. If the pre-upgrade check fails (e.g. because of the HTTP_PROXY env var is set or otherwise), no upgrade nor satellite-installer is run. So the satellite-installer unset/set functionality is unreachable and thus appears redundant to me.

Is any of the assumptions wrong?